### DISCUSSION TITLE
Visual Editor like gRPC UI

### DISCUSSION THREAD

**Discussion by mishushakov** [getezy/ezy#29] [Feature requests]
https://github.com/fullstorydev/grpcui

**Comment by notmedia** [comment]
Interesting idea, at the start of this project I had an idea to use [react-json-view](https://www.npmjs.com/package/react-json-view) library for response view.

But form for request and metadata is really interesting idea, I will think about this. Thank you!

**Reply by notmedia** [reply]
Also related to json view - https://jsonhero.io/
https://jsoncrack.com/editor
